### PR TITLE
Ft profile update

### PR DESCRIPTION
**Pull request by Kalimbak**
What does this PR do?
       Creates or updates the user's profile

Description of Task to be completed?
     As **operator or driver**, I want to be able to update information on my profile, so that others could see updated information about me.
    Appropriate error messages are displayed on unsuccessful attempts

  How should this be manually tested?
        git pull this branch
        run npm install
        check env.example file and add JWT_SECRET and JWT_EXPIRES_IN to your .env file
        Do migrations with npm run migrate
        run npx sequelize-cli db:seed:all to create pre-configured credentials for seeders
        run npm run dev
Test these routes 
        POST /api/v1/profile/update/:id in POSTMAN or any other platform for using API

### CODE REVIEW

**Review by Angelus123 — APPROVED** [review]

File: src/documentation/drivers&operators.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+          description: 'Create user',
+          required: true,
+          schema: {
+            $ref: '#/definitions/Register',
+          },
+        },
+        // {
+        //   name: 'Accept-Language',
+        //   in: 'header',
+        //   description: 'the Accept-Language',
```
Comment (tjeaneric) [inline src/documentation/drivers&operators.js]: remove unnecessary

**Review by tjeaneric — APPROVED** [review]

File: src/documentation/drivers&operators.js (hunk 1 above)
Comment (Kalimbak) [reply src/documentation/drivers&operators.js]: alright

**Review by kwizeradev — APPROVED** [review]

File: Procfile [hunk 2]
```diff
@@ -1 +1,11 @@
...
+<<<<<<< HEAD
+=======
+<<<<<<< HEAD
+web: npm start
+=======
+web: npm start
+>>>>>>> 3ede931... crud route tests
+>>>>>>> added Get Post Delete Put for routes
+=======
+>>>>>>> fixed
```
Comment (furebo) [inline Procfile]: Remove unnecessary lines in Procfile

File: __test__/app.test.js [hunk 3]
```diff
@@ -1,11 +1,15 @@
+<<<<<<< HEAD
 //During the test the env variable is set to test
 
 //Require the dev-dependencies
+=======
+>>>>>>> 3ede931... crud route tests
```
Comment (furebo) [inline __test__/app.test.js]: Please remove these lines

File: __test__/app.test.js [hunk 4]
```diff
@@ -15,18 +19,24 @@ describe('Root testing: welcome', () => {
...
         chai.expect(res.statusCode).to.equal(200);
         chai.expect(res.body).to.have.property('message');
+<<<<<<< HEAD
         // res.should.have.status(200);
         // res.body.should.have
         //   .property('message')
         //   .eql('Welcome to Dominators-Phantom-API!');
         // done();
+=======
+>>>>>>> 3ede931... crud route tests
```
Comment (furebo) [inline __test__/app.test.js]: Remove these lines also

**Review by furebo — CHANGES_REQUESTED** [review]
Please describe well your PR, squash the commits to one commit and work on requested changes.

File: Procfile (hunk 2 above)
Comment (furebo) [inline Procfile]: Please remove those Unnecessary lines in the file and remember to squash the commits to one commit

**Review by furebo — CHANGES_REQUESTED** [review]

**Review by furebo — APPROVED** [review]

**Pull request merged**